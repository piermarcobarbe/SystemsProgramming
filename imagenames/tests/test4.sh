#!/bin/sh

exec "$1" "tests/extensions3" < tests/test4.in
