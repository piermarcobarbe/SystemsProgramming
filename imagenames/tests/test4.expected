file.1.321
file.1.322
file.1.323
file.1.324
file.1.325
file.1.326
file.1.327
file.1.328
file.1.329
file.1.330
file.1.331
file.1.332
file.1.333
file.1.334
file.1.335
file.1.336
file.1.337
file.1.338
file.1.339
file.1.340
file.1.341
file.1.342
file.1.343
file.1.344
file.1.345
file.3.321
file.3.322
file.3.323
file.3.324
file.3.325
file.3.326
file.3.327
file.3.328
file.3.329
file.3.330
file.3.331
file.3.332
file.3.333
file.3.334
file.3.335
file.3.336
file.3.337
file.3.338
file.3.339
file.3.340
file.3.341
file.3.342
file.3.343
file.3.344
file.3.345
file.5.321
file.5.322
file.5.323
file.5.324
file.5.325
file.5.326
file.5.327
file.5.328
file.5.329
file.5.330
file.5.331
file.5.332
file.5.333
file.5.334
file.5.335
file.5.336
file.5.337
file.5.338
file.5.339
file.5.340
file.5.341
file.5.342
file.5.343
file.5.344
file.5.345
file.7.321
file.7.322
file.7.323
file.7.324
file.7.325
file.7.326
file.7.327
file.7.328
file.7.329
file.7.330
file.7.331
file.7.332
file.7.333
file.7.334
file.7.335
file.7.336
file.7.337
file.7.338
file.7.339
file.7.340
file.7.341
file.7.342
file.7.343
file.7.344
file.7.345
file.9.321
file.9.322
file.9.323
file.9.324
file.9.325
file.9.326
file.9.327
file.9.328
file.9.329
file.9.330
file.9.331
file.9.332
file.9.333
file.9.334
file.9.335
file.9.336
file.9.337
file.9.338
file.9.339
file.9.340
file.9.341
file.9.342
file.9.343
file.9.344
file.9.345
file.11.321
file.11.322
file.11.323
file.11.324
file.11.325
file.11.326
file.11.327
file.11.328
file.11.329
file.11.330
file.11.331
file.11.332
file.11.333
file.11.334
file.11.335
file.11.336
file.11.337
file.11.338
file.11.339
file.11.340
file.11.341
file.11.342
file.11.343
file.11.344
file.11.345
file.13.321
file.13.322
file.13.323
file.13.324
file.13.325
file.13.326
file.13.327
file.13.328
file.13.329
file.13.330
file.13.331
file.13.332
file.13.333
file.13.334
file.13.335
file.13.336
file.13.337
file.13.338
file.13.339
file.13.340
file.13.341
file.13.342
file.13.343
file.13.344
file.13.345
file.15.321
file.15.322
file.15.323
file.15.324
file.15.325
file.15.326
file.15.327
file.15.328
file.15.329
file.15.330
file.15.331
file.15.332
file.15.333
file.15.334
file.15.335
file.15.336
file.15.337
file.15.338
file.15.339
file.15.340
file.15.341
file.15.342
file.15.343
file.15.344
file.15.345
file.17.321
file.17.322
file.17.323
file.17.324
file.17.325
file.17.326
file.17.327
file.17.328
file.17.329
file.17.330
file.17.331
file.17.332
file.17.333
file.17.334
file.17.335
file.17.336
file.17.337
file.17.338
file.17.339
file.17.340
file.17.341
file.17.342
file.17.343
file.17.344
file.17.345
file.19.321
file.19.322
file.19.323
file.19.324
file.19.325
file.19.326
file.19.327
file.19.328
file.19.329
file.19.330
file.19.331
file.19.332
file.19.333
file.19.334
file.19.335
file.19.336
file.19.337
file.19.338
file.19.339
file.19.340
file.19.341
file.19.342
file.19.343
file.19.344
file.19.345
file.21.321
file.21.322
file.21.323
file.21.324
file.21.325
file.21.326
file.21.327
file.21.328
file.21.329
file.21.330
file.21.331
file.21.332
file.21.333
file.21.334
file.21.335
file.21.336
file.21.337
file.21.338
file.21.339
file.21.340
file.21.341
file.21.342
file.21.343
file.21.344
file.21.345
file.23.321
file.23.322
file.23.323
file.23.324
file.23.325
file.23.326
file.23.327
file.23.328
file.23.329
file.23.330
file.23.331
file.23.332
file.23.333
file.23.334
file.23.335
file.23.336
file.23.337
file.23.338
file.23.339
file.23.340
file.23.341
file.23.342
file.23.343
file.23.344
file.23.345
file.25.321
file.25.322
file.25.323
file.25.324
file.25.325
file.25.326
file.25.327
file.25.328
file.25.329
file.25.330
file.25.331
file.25.332
file.25.333
file.25.334
file.25.335
file.25.336
file.25.337
file.25.338
file.25.339
file.25.340
file.25.341
file.25.342
file.25.343
file.25.344
file.25.345
file.27.321
file.27.322
file.27.323
file.27.324
file.27.325
file.27.326
file.27.327
file.27.328
file.27.329
file.27.330
file.27.331
file.27.332
file.27.333
file.27.334
file.27.335
file.27.336
file.27.337
file.27.338
file.27.339
file.27.340
file.27.341
file.27.342
file.27.343
file.27.344
file.27.345
file.29.321
file.29.322
file.29.323
file.29.324
file.29.325
file.29.326
file.29.327
file.29.328
file.29.329
file.29.330
file.29.331
file.29.332
file.29.333
file.29.334
file.29.335
file.29.336
file.29.337
file.29.338
file.29.339
file.29.340
file.29.341
file.29.342
file.29.343
file.29.344
file.29.345
file.31.321
file.31.322
file.31.323
file.31.324
file.31.325
file.31.326
file.31.327
file.31.328
file.31.329
file.31.330
file.31.331
file.31.332
file.31.333
file.31.334
file.31.335
file.31.336
file.31.337
file.31.338
file.31.339
file.31.340
file.31.341
file.31.342
file.31.343
file.31.344
file.31.345
file.33.321
file.33.322
file.33.323
file.33.324
file.33.325
file.33.326
file.33.327
file.33.328
file.33.329
file.33.330
file.33.331
file.33.332
file.33.333
file.33.334
file.33.335
file.33.336
file.33.337
file.33.338
file.33.339
file.33.340
file.33.341
file.33.342
file.33.343
file.33.344
file.33.345
file.35.321
file.35.322
file.35.323
file.35.324
file.35.325
file.35.326
file.35.327
file.35.328
file.35.329
file.35.330
file.35.331
file.35.332
file.35.333
file.35.334
file.35.335
file.35.336
file.35.337
file.35.338
file.35.339
file.35.340
file.35.341
file.35.342
file.35.343
file.35.344
file.35.345
file.37.321
file.37.322
file.37.323
file.37.324
file.37.325
file.37.326
file.37.327
file.37.328
file.37.329
file.37.330
file.37.331
file.37.332
file.37.333
file.37.334
file.37.335
file.37.336
file.37.337
file.37.338
file.37.339
file.37.340
file.37.341
file.37.342
file.37.343
file.37.344
file.37.345
file.39.321
file.39.322
file.39.323
file.39.324
file.39.325
file.39.326
file.39.327
file.39.328
file.39.329
file.39.330
file.39.331
file.39.332
file.39.333
file.39.334
file.39.335
file.39.336
file.39.337
file.39.338
file.39.339
file.39.340
file.39.341
file.39.342
file.39.343
file.39.344
file.39.345
file.41.321
file.41.322
file.41.323
file.41.324
file.41.325
file.41.326
file.41.327
file.41.328
file.41.329
file.41.330
file.41.331
file.41.332
file.41.333
file.41.334
file.41.335
file.41.336
file.41.337
file.41.338
file.41.339
file.41.340
file.41.341
file.41.342
file.41.343
file.41.344
file.41.345
file.43.321
file.43.322
file.43.323
file.43.324
file.43.325
file.43.326
file.43.327
file.43.328
file.43.329
file.43.330
file.43.331
file.43.332
file.43.333
file.43.334
file.43.335
file.43.336
file.43.337
file.43.338
file.43.339
file.43.340
file.43.341
file.43.342
file.43.343
file.43.344
file.43.345
file.45.321
file.45.322
file.45.323
file.45.324
file.45.325
file.45.326
file.45.327
file.45.328
file.45.329
file.45.330
file.45.331
file.45.332
file.45.333
file.45.334
file.45.335
file.45.336
file.45.337
file.45.338
file.45.339
file.45.340
file.45.341
file.45.342
file.45.343
file.45.344
file.45.345
file.47.321
file.47.322
file.47.323
file.47.324
file.47.325
file.47.326
file.47.327
file.47.328
file.47.329
file.47.330
file.47.331
file.47.332
file.47.333
file.47.334
file.47.335
file.47.336
file.47.337
file.47.338
file.47.339
file.47.340
file.47.341
file.47.342
file.47.343
file.47.344
file.47.345
file.49.321
file.49.322
file.49.323
file.49.324
file.49.325
file.49.326
file.49.327
file.49.328
file.49.329
file.49.330
file.49.331
file.49.332
file.49.333
file.49.334
file.49.335
file.49.336
file.49.337
file.49.338
file.49.339
file.49.340
file.49.341
file.49.342
file.49.343
file.49.344
file.49.345
file.51.321
file.51.322
file.51.323
file.51.324
file.51.325
file.51.326
file.51.327
file.51.328
file.51.329
file.51.330
file.51.331
file.51.332
file.51.333
file.51.334
file.51.335
file.51.336
file.51.337
file.51.338
file.51.339
file.51.340
file.51.341
file.51.342
file.51.343
file.51.344
file.51.345
file.53.321
file.53.322
file.53.323
file.53.324
file.53.325
file.53.326
file.53.327
file.53.328
file.53.329
file.53.330
file.53.331
file.53.332
file.53.333
file.53.334
file.53.335
file.53.336
file.53.337
file.53.338
file.53.339
file.53.340
file.53.341
file.53.342
file.53.343
file.53.344
file.53.345
file.55.321
file.55.322
file.55.323
file.55.324
file.55.325
file.55.326
file.55.327
file.55.328
file.55.329
file.55.330
file.55.331
file.55.332
file.55.333
file.55.334
file.55.335
file.55.336
file.55.337
file.55.338
file.55.339
file.55.340
file.55.341
file.55.342
file.55.343
file.55.344
file.55.345
file.57.321
file.57.322
file.57.323
file.57.324
file.57.325
file.57.326
file.57.327
file.57.328
file.57.329
file.57.330
file.57.331
file.57.332
file.57.333
file.57.334
file.57.335
file.57.336
file.57.337
file.57.338
file.57.339
file.57.340
file.57.341
file.57.342
file.57.343
file.57.344
file.57.345
file.59.321
file.59.322
file.59.323
file.59.324
file.59.325
file.59.326
file.59.327
file.59.328
file.59.329
file.59.330
file.59.331
file.59.332
file.59.333
file.59.334
file.59.335
file.59.336
file.59.337
file.59.338
file.59.339
file.59.340
file.59.341
file.59.342
file.59.343
file.59.344
file.59.345
file.61.321
file.61.322
file.61.323
file.61.324
file.61.325
file.61.326
file.61.327
file.61.328
file.61.329
file.61.330
file.61.331
file.61.332
file.61.333
file.61.334
file.61.335
file.61.336
file.61.337
file.61.338
file.61.339
file.61.340
file.61.341
file.61.342
file.61.343
file.61.344
file.61.345
file.63.321
file.63.322
file.63.323
file.63.324
file.63.325
file.63.326
file.63.327
file.63.328
file.63.329
file.63.330
file.63.331
file.63.332
file.63.333
file.63.334
file.63.335
file.63.336
file.63.337
file.63.338
file.63.339
file.63.340
file.63.341
file.63.342
file.63.343
file.63.344
file.63.345
file.65.321
file.65.322
file.65.323
file.65.324
file.65.325
file.65.326
file.65.327
file.65.328
file.65.329
file.65.330
file.65.331
file.65.332
file.65.333
file.65.334
file.65.335
file.65.336
file.65.337
file.65.338
file.65.339
file.65.340
file.65.341
file.65.342
file.65.343
file.65.344
file.65.345
file.67.321
file.67.322
file.67.323
file.67.324
file.67.325
file.67.326
file.67.327
file.67.328
file.67.329
file.67.330
file.67.331
file.67.332
file.67.333
file.67.334
file.67.335
file.67.336
file.67.337
file.67.338
file.67.339
file.67.340
file.67.341
file.67.342
file.67.343
file.67.344
file.67.345
file.69.321
file.69.322
file.69.323
file.69.324
file.69.325
file.69.326
file.69.327
file.69.328
file.69.329
file.69.330
file.69.331
file.69.332
file.69.333
file.69.334
file.69.335
file.69.336
file.69.337
file.69.338
file.69.339
file.69.340
file.69.341
file.69.342
file.69.343
file.69.344
file.69.345
file.71.321
file.71.322
file.71.323
file.71.324
file.71.325
file.71.326
file.71.327
file.71.328
file.71.329
file.71.330
file.71.331
file.71.332
file.71.333
file.71.334
file.71.335
file.71.336
file.71.337
file.71.338
file.71.339
file.71.340
file.71.341
file.71.342
file.71.343
file.71.344
file.71.345
file.73.321
file.73.322
file.73.323
file.73.324
file.73.325
file.73.326
file.73.327
file.73.328
file.73.329
file.73.330
file.73.331
file.73.332
file.73.333
file.73.334
file.73.335
file.73.336
file.73.337
file.73.338
file.73.339
file.73.340
file.73.341
file.73.342
file.73.343
file.73.344
file.73.345
file.75.321
file.75.322
file.75.323
file.75.324
file.75.325
file.75.326
file.75.327
file.75.328
file.75.329
file.75.330
file.75.331
file.75.332
file.75.333
file.75.334
file.75.335
file.75.336
file.75.337
file.75.338
file.75.339
file.75.340
file.75.341
file.75.342
file.75.343
file.75.344
file.75.345
file.77.321
file.77.322
file.77.323
file.77.324
file.77.325
file.77.326
file.77.327
file.77.328
file.77.329
file.77.330
file.77.331
file.77.332
file.77.333
file.77.334
file.77.335
file.77.336
file.77.337
file.77.338
file.77.339
file.77.340
file.77.341
file.77.342
file.77.343
file.77.344
file.77.345
file.79.321
file.79.322
file.79.323
file.79.324
file.79.325
file.79.326
file.79.327
file.79.328
file.79.329
file.79.330
file.79.331
file.79.332
file.79.333
file.79.334
file.79.335
file.79.336
file.79.337
file.79.338
file.79.339
file.79.340
file.79.341
file.79.342
file.79.343
file.79.344
file.79.345
file.81.321
file.81.322
file.81.323
file.81.324
file.81.325
file.81.326
file.81.327
file.81.328
file.81.329
file.81.330
file.81.331
file.81.332
file.81.333
file.81.334
file.81.335
file.81.336
file.81.337
file.81.338
file.81.339
file.81.340
file.81.341
file.81.342
file.81.343
file.81.344
file.81.345
file.83.321
file.83.322
file.83.323
file.83.324
file.83.325
file.83.326
file.83.327
file.83.328
file.83.329
file.83.330
file.83.331
file.83.332
file.83.333
file.83.334
file.83.335
file.83.336
file.83.337
file.83.338
file.83.339
file.83.340
file.83.341
file.83.342
file.83.343
file.83.344
file.83.345
file.85.321
file.85.322
file.85.323
file.85.324
file.85.325
file.85.326
file.85.327
file.85.328
file.85.329
file.85.330
file.85.331
file.85.332
file.85.333
file.85.334
file.85.335
file.85.336
file.85.337
file.85.338
file.85.339
file.85.340
file.85.341
file.85.342
file.85.343
file.85.344
file.85.345
file.87.321
file.87.322
file.87.323
file.87.324
file.87.325
file.87.326
file.87.327
file.87.328
file.87.329
file.87.330
file.87.331
file.87.332
file.87.333
file.87.334
file.87.335
file.87.336
file.87.337
file.87.338
file.87.339
file.87.340
file.87.341
file.87.342
file.87.343
file.87.344
file.87.345
file.89.321
file.89.322
file.89.323
file.89.324
file.89.325
file.89.326
file.89.327
file.89.328
file.89.329
file.89.330
file.89.331
file.89.332
file.89.333
file.89.334
file.89.335
file.89.336
file.89.337
file.89.338
file.89.339
file.89.340
file.89.341
file.89.342
file.89.343
file.89.344
file.89.345
file.91.321
file.91.322
file.91.323
file.91.324
file.91.325
file.91.326
file.91.327
file.91.328
file.91.329
file.91.330
file.91.331
file.91.332
file.91.333
file.91.334
file.91.335
file.91.336
file.91.337
file.91.338
file.91.339
file.91.340
file.91.341
file.91.342
file.91.343
file.91.344
file.91.345
file.93.321
file.93.322
file.93.323
file.93.324
file.93.325
file.93.326
file.93.327
file.93.328
file.93.329
file.93.330
file.93.331
file.93.332
file.93.333
file.93.334
file.93.335
file.93.336
file.93.337
file.93.338
file.93.339
file.93.340
file.93.341
file.93.342
file.93.343
file.93.344
file.93.345
file.95.321
file.95.322
file.95.323
file.95.324
file.95.325
file.95.326
file.95.327
file.95.328
file.95.329
file.95.330
file.95.331
file.95.332
file.95.333
file.95.334
file.95.335
file.95.336
file.95.337
file.95.338
file.95.339
file.95.340
file.95.341
file.95.342
file.95.343
file.95.344
file.95.345
file.97.321
file.97.322
file.97.323
file.97.324
file.97.325
file.97.326
file.97.327
file.97.328
file.97.329
file.97.330
file.97.331
file.97.332
file.97.333
file.97.334
file.97.335
file.97.336
file.97.337
file.97.338
file.97.339
file.97.340
file.97.341
file.97.342
file.97.343
file.97.344
file.97.345
file.99.321
file.99.322
file.99.323
file.99.324
file.99.325
file.99.326
file.99.327
file.99.328
file.99.329
file.99.330
file.99.331
file.99.332
file.99.333
file.99.334
file.99.335
file.99.336
file.99.337
file.99.338
file.99.339
file.99.340
file.99.341
file.99.342
file.99.343
file.99.344
file.99.345
